### PR TITLE
refactor: split nodes columns file, use nodes api, update types

### PR DESCRIPTION
**Pull request by artemmufazalov**
### Test Status: <span style="color: red;">❌ FAILED</span>
📊 [Full Report](https://ydb-platform.github.io/ydb-embedded-ui/1310/)

| Total | Passed | Failed | Flaky | Skipped |
|:-----:|:------:|:------:|:-----:|:-------:|
| 124 | 116 | 2 | 6 | 0 |

### Bundle Size: 🔺
Current: 78.98 MB | Main: 78.97 MB
Diff: +0.01 MB (0.01%)

⚠️ Bundle size increased. Please review.

<details>
<summary>ℹ️ CI Information</summary>

- Test recordings for failed tests are available in the full report.
- Bundle size is measured for the entire 'dist' directory.
- 📊 indicates links to detailed reports.
- 🔺 indicates increase, 🔽 decrease, and ✅ no change in bundle size.
</details>

### CODE REVIEW

File: src/containers/Tenant/Diagnostics/TenantOverview/TenantCpu/TopNodesByCpu.tsx [hunk 1]
```diff
@@ -22,13 +22,14 @@ export function TopNodesByCpu({tenantName, additionalNodesProps}: TopNodesByCpuP
     const [autoRefreshInterval] = useAutoRefreshInterval();
     const columns = getTopNodesByCpuColumns(additionalNodesProps?.getNodeRef);
 
-    const {currentData, isFetching, error} = topNodesApi.useGetTopNodesQuery(
-        {tenant: tenantName, sortValue: 'CPU'},
+    const {currentData, isFetching, error} = nodesApi.useGetNodesQuery(
```
Comment (artemmufazalov) [inline src/containers/Tenant/Diagnostics/TenantOverview/TenantCpu/TopNodesByCpu.tsx:25]: Made all nodes tables use the same api. This way they will have single cache entry and single prepare function

File: src/containers/Tenant/Diagnostics/TenantOverview/TenantCpu/utils.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+    nodes.sort((node1, node2) => {
+        const node1SortValue = getNodeMaxPoolUsage(node1.PoolStats);
+        const node2SortValue = getNodeMaxPoolUsage(node2.PoolStats);
+
+        return node2SortValue - node1SortValue;
+    });
+
+    return nodes.slice(0, TENANT_OVERVIEW_TABLES_LIMIT);
+}
```
Comment (artemmufazalov) [inline src/containers/Tenant/Diagnostics/TenantOverview/TenantCpu/utils.ts:25]: In the latest `viewer/nodes` updates some sort params was removed from backend. Implemented additional sorting and filtered on frontend to ensure correct results

File: src/types/api/nodes.ts [hunk 3]
```diff
@@ -133,3 +133,79 @@ enum EConfigState {
...
+    | 'DiskSpaceUsage'
+    | 'SubDomainKey'
+    | 'DisconnectTime'
+    | 'Database';
+
+export type NodesSortValue = 'NodeId' | 'Host' | 'NodeName' | 'DC' | 'Rack' | 'Version' | 'Uptime';
+
+export type NodesSort = `-${NodesSortValue}` | `+${NodesSortValue}`;
+
+export interface NodesRequestParams {
```
Comment (artemmufazalov) [inline src/types/api/nodes.ts:180]: Described all params, that are currently available on backend

File: src/containers/Nodes/columns/constants.ts [hunk 4]
```diff
@@ -0,0 +1,69 @@
+import type {ValueOf} from '../../../types/common';
+
+import i18n from './i18n';
+
+export const NODES_COLUMNS_WIDTH_LS_KEY = 'nodesTableColumnsWidth';
+
+export const NODES_COLUMNS_IDS = {
```
Comment (artemmufazalov) [inline src/containers/Nodes/columns/constants.ts:7]: Just prepared columns files for further updates

File: src/containers/Tenant/Diagnostics/TenantOverview/TenantCpu/TopNodesByLoad.tsx [hunk 5]
```diff
@@ -22,13 +22,14 @@ export function TopNodesByLoad({tenantName, additionalNodesProps}: TopNodesByLoa
...
-        {tenant: tenantName, sortValue: 'LoadAverage'},
+    const {currentData, isFetching, error} = nodesApi.useGetNodesQuery(
+        {tenant: tenantName, type: 'any'},
         {pollingInterval: autoRefreshInterval},
     );
 
     const loading = isFetching && currentData === undefined;
-    const topNodes = currentData;
+
+    const topNodes = prepareTopNodesByLoad(currentData);
```
Comment (astandrik) [inline src/containers/Tenant/Diagnostics/TenantOverview/TenantCpu/TopNodesByLoad.tsx:32]: would React.memo be a good idea here?

File: src/containers/Storage/utils/index.ts [hunk 6]
```diff
@@ -41,7 +40,7 @@ export const getUsageSeverityForEntityStatus = generateEvaluator(80, 85, [
 ]);
 
 export const defaultSortNode: NodesSortParams = {
-    sortValue: NODES_SORT_VALUES.NodeId,
+    sortValue: 'NodeId',
```
Comment (astandrik) [inline src/containers/Storage/utils/index.ts:43]: NODES_COLUMNS_IDS.NodeId ?

**Review by astandrik — CHANGES_REQUESTED** [review]

**Review by astandrik — APPROVED** [review]

**Pull request merged**